### DISCUSSION TITLE
Isn't it okay to remove simple_webhook_url and simple_proxy?

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#743] [Ideas]
As far as I read the message in the warning log, I thought that the code related to simple_webhook_url and simple_proxy could be deleted.

```python
            elastalert_logger.warning(
                '"simple" alerter has been renamed "post" and comptability may be removed in a future release.')
```

https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml#L287

```yaml
  ### Simple
  simple_webhook_url: *arrayOfString
  simple_proxy: {type: string}
```

L554 - 564
https://github.com/jertel/elastalert2/blob/master/elastalert/loaders.py#L554

```python
    @staticmethod
    def adjust_deprecated_values(rule):
        # From rename of simple HTTP alerter
        if rule.get('type') == 'simple':
            rule['type'] = 'post'
            if 'simple_proxy' in rule:
                rule['http_post_proxy'] = rule['simple_proxy']
            if 'simple_webhook_url' in rule:
                rule['http_post_url'] = rule['simple_webhook_url']
            elastalert_logger.warning(
                '"simple" alerter has been renamed "post" and comptability may be removed in a future release.')
```

L296
https://github.com/jertel/elastalert2/blob/master/elastalert/loaders.py#L296

```python
        self.adjust_deprecated_values(rule)
```

**Comment by jertel** [comment]
Yes, we could remove those schema defs and the code but it will be a breaking change for people that still use the old names, so we'll need to identify that in the changelog.

**Reply by nsano-rururu** [reply]
>Yes, we could remove those schema defs and the code but it will be a breaking change for people that still use the old names, so we'll need to identify that in the changelog.

When issuing a pull request, it should be described in the Breaking changes of the change log.

**Reply by nsano-rururu** [reply]
I submitted a pull request
https://github.com/jertel/elastalert2/pull/793
